### PR TITLE
Added a page on JADE 2 as requested in #1034

### PR DESCRIPTION
**Pull request by twinkarma**

### CODE REVIEW

File: jade2.rst [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+**More details on the system coming soon.**
+
+Hardware
+^^^^^^^^
+
+* 63 Nodes of DGX MAX-Q, each with:
+    * 8x Nvidia V100 32GB GPUs
+    * 512GB RAM
+    * 70TB
```
Comment (willfurnass) [inline jade2.rst:26]: 70TB local storage per node?

Comment (twinkarma) [reply jade2.rst:26]: Oops I did a copy instead of cut

File: jade2.rst [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+**More details on the system coming soon.**
+
+Hardware
+^^^^^^^^
+
+* 63 Nodes of DGX MAX-Q, each with:
+    * 8x Nvidia V100 32GB GPUs
+    * 512GB RAM
+    * 70TB 
+* 70TB DDN AI400 NVMe storage
```
Comment (willfurnass) [inline jade2.rst:26]: This suggests it's NVMe storage array(s) that run a parallel filesystem and are accessed over the EDR Infiniband cluster fabric - is this right?

File: jade2.rst [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+* 1PB Lustre disk storage
+* EDR infiniband interconnect
```
Comment (willfurnass) [inline jade2.rst:28]: For connections to storage or for inter-node comms too?

File: jade2.rst [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+Hardware
+^^^^^^^^
+
+* 63 Nodes of DGX MAX-Q, each with:
+    * 8x Nvidia V100 32GB GPUs
+    * 512GB RAM
+    * 70TB 
+* 70TB DDN AI400 NVMe storage
+* 1PB Lustre disk storage
```
Comment (willfurnass) [inline jade2.rst:27]: Worth noting that Lustre is a shared filesystem that (presumably) can be accessed by all nodes in the cluster.

File: jade2.rst (hunk 3 above)
Comment (twinkarma) [reply jade2.rst:28]: It should be infiniband for everything.

Comment (willfurnass) [reply jade2.rst:28]: So multi-node jobs will be permitted with JADE II?

File: jade2.rst (hunk 4 above)
Comment (twinkarma) [reply jade2.rst:27]: Yes the 70TB NVMe and 1PB storage should both be network storage. One for fast read/write and one for normal storage/archive.

Comment (willfurnass) [reply jade2.rst:27]: Interesting.  I think that comment would be worth adding to this page.

File: jade2.rst (hunk 3 above)
Comment (twinkarma) [reply jade2.rst:28]: I've heard it mentioned that its on the cards but will confirm again.

Comment (willfurnass) [reply jade2.rst:28]: I guess it's probably wise not to mention the possibility of  multi-node jobs here until that's been confirmed.

**Comment by willfurnass** [comment]
@twinkarma OK to merge?

**Comment by twinkarma** [comment]
Yes let's merge it in then, I won't be getting any new info in the nearish future.

**Comment by willfurnass** [comment]
Thanks @twinkarma !

**Pull request merged**